<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server_Form</class>
 <widget class="QDialog" name="Server_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lab_id">
         <property name="text">
          <string>服务端ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_ip">
         <property name="text">
          <string>服务端IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_port">
         <property name="text">
          <string>服务端端口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_dataType">
         <property name="text">
          <string>数据类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_publicAddr">
         <property name="text">
          <string>公共地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_describe">
         <property name="text">
          <string>描述</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_id"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ip"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_port"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_dataType"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_publicAddr"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_describe"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
